bot/seasons/evergreen/battleship.py

```diff
@@ -0,0 +1,444 @@
+    @commands.guild_only()
+    async def battleship(self, ctx: commands.Context) -> None:
+        """
+        Play a game of Battleship with someone else!
+
+        This will set up a message waiting for someone else to react and play along.
+        The game takes place entirely in DMs.
+        Make sure you have your DMs open so that the bot can message you.
+        """
+        if self.already_playing(ctx.author):
+            return await ctx.send("You're already playing a game!")
+
+        if ctx.author in self.waiting:
+            return await ctx.send("You've already sent out a request for a player 2")
+
+        announcement = await ctx.send(
+            "**Battleship**: A new game is about to start!\n"
+            f"Press {HAND_RAISED_EMOJI} to play against {ctx.author.mention}!\n"
+            f"(Cancel the game with {CROSS_EMOJI}.)"
+        )
+        self.waiting.append(ctx.author)
+        await announcement.add_reaction(HAND_RAISED_EMOJI)
+        await announcement.add_reaction(CROSS_EMOJI)
+
+        try:
+            reaction, user = await self.bot.wait_for(
+                "reaction_add",
+                check=partial(self.predicate, ctx, announcement),
+                timeout=60.0
+            )
+        except asyncio.TimeoutError:
+            self.waiting.remove(ctx.author)
+            await announcement.delete()
+            return await ctx.send(f"{ctx.author.mention} Seems like there's no one here to play...")
+
+        if str(reaction.emoji) == CROSS_EMOJI:
+            self.waiting.remove(ctx.author)
+            await announcement.delete()
+            return await ctx.send(f"{ctx.author.mention} Game cancelled.")
+
+        await announcement.delete()
+        self.waiting.remove(ctx.author)
+        if self.already_playing(ctx.author):
+            return
+        try:
+            game = Game(self.bot, ctx.channel, ctx.author, user)
+            self.games.append(game)
+            await game.start_game()
+            self.games.remove(game)
+        except discord.Forbidden:
+            await ctx.send(
+                f"{ctx.author.mention} {user.mention} "
+                "Game failed. This is likely due to you not having your DMs open. Check and try again."
+            )
+            self.games.remove(game)
+        except Exception:
+            # End the game in the event of an unforseen error so the players aren't stuck in a game
+            await ctx.send(f"{ctx.author.mention} {user.mention} An error occurred. Game failed")
+            self.games.remove(game)
+            raise
+
+    @battleship.command(name="ships", aliases=["boats"])
+    async def battleship_ships(self, ctx: commands.Context) -> None:
+        """Lists the ships that are found on the battleship grid."""
+        embed = discord.Embed(colour=Colours.blue)
+        embed.add_field(name="Name", value="Carrier\nBattleship\nCruiser\nSubmarine\nDestroyer")
+        embed.add_field(name="Size", value="5\n4\n3\n3\n2")
```

Comment:
Maybe use `SHIPS` constant joining instead hard-coded strings? But this is only thing. Everything other is good.
```suggestion
        embed.add_field(name="Name", value="\n".join(SHIPS))
        embed.add_field(name="Size", value="\n".join(str(size) for size in SHIPS.values()))
```